application/LaunchController.cpp

```diff
@@ -215,7 +218,41 @@ void LaunchController::launchInstance()
     connect(m_launcher.get(), &LaunchTask::failed, this,  &LaunchController::onFailed);
     connect(m_launcher.get(), &LaunchTask::requestProgress, this, &LaunchController::onProgressRequested);
 
+    // Prepend Server Status
+    QStringList servers = {"authserver.mojang.com", "session.minecraft.net", "textures.minecraft.net", "api.mojang.com"};
+    QString resolved_servers = "";
+    QHostInfo host_info;
 
+    for(QString server : servers) {
+        host_info = QHostInfo::fromName(server);
+        resolved_servers = resolved_servers + server + " resolves to:";
+        if(host_info.addresses().size() > 0) {
+            for(QHostAddress address : host_info.addresses()) {
+                resolved_servers = resolved_servers + "\n" + address.toString();
```

Comment:
IMO it would be nicer if the resolved servers were comma-separated.

Reply:
could you explain what you mean, i dont quite understand in what way seperated, dont want to implement only for it to be not the way you thought off :D

Reply:
Sure - take the following example:

```
textures.minecraft.net resolves to:
13.224.195.6
13.224.195.63
13.224.195.13
13.224.195.42
```

would become something like

```
textures.minecraft.net resolves to:
    [13.224.195.6, 13.224.195.63, 13.224.195.13, 13.224.195.42]
```

Reply:
I can do that, that was actually almost what i planned to do the first time

Reply:
with tab?

Reply:
I think it looks clearer with a tab personally.